backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -187,7 +190,12 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
 
     globFiles.map(globFiles =>
     s"""|#!/bin/bash
-        |tmpDir=$$(mktemp -d $cwd/tmp.XXXXXX)
+        |tmpDir=$$(
+        |  set -e
+        |  cd $cwd
+        |  tmpDir="$temporaryDirectory"
+        |  echo "$$tmpDir"
```

Comment:
does this print to stdout ?

Reply:
In `$()`, and the deprecated ` `` ` syntax, stdout is captured while stderr is printed.

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03